### DISCUSSION TITLE
Output publication as MOBI file

### DISCUSSION THREAD

**Discussion by junedtan** [thegetty/quire#964] [Get Help]
While I understand that "quire pdf" and "quire epub" are two of the output formats available, how can I output a publication as mobi? Tried "quire mobi" with "unknown option" error, and also looking at v1 docs and found nothing about it.

I also tried Calibre (https://calibre-ebook.com/) to convert resulting pdf to mobi, but these output errors happen: 1) images are randomly displayed as blank (black rectangle), 2) numbers are displayed as boxes (and gone after switching to default fonts like Times New Roman or Arial), 3) white spaces between some paragraphs where there should be none, etc. I'd say it might be Calibre's limitations, so I'd like to understand the workflow and tools used to generate mobi file like this (https://www.getty.edu/publications/cva10/downloads/Tsiafakis_CVA10.mobi). 

Thank you in advance.

**Comment by Erin-Cecele** [comment]
Hello @junedtan, Amazon officially retired the MOBI format and replaced it with EPUB in August 2022. For that reason, Quire v1 stopped outputting MOBI versions and solely uses EPUB now. That said, you could try working with v0 which did produce MOBI (https://quire.getty.edu/docs-v0/multiformat-output/). However, v0 is out of date and lacking both newer features and technical support.

**Comment by junedtan** [comment]
@Erin-Cecele that's amazing info, thank you very much! Since I'm using v1 I guess I don't need to worry about it now.
